src/marionette.rs

```diff
@@ -793,7 +793,7 @@ impl MarionetteCommand {
             },
             DeleteSession => {
                 let mut body = BTreeMap::new();
-                body.insert("flags".to_owned(), vec!["eForceQuit".to_json()].to_json());
+                body.insert("flags".to_owned(), Json::Array(vec![]));
```

Comment:
I would change this to not send an array at all:

```rust
DeleteSession => (Some("quitApplication", None),
```

This would work with existing Firefoxen and the new behaviour where we fall back to `eAttemptQuit` if `flags` is undefined in https://bugzilla.mozilla.org/show_bug.cgi?id=1337743.